spec/models/guide_spec.rb

```diff
@@ -129,20 +129,29 @@
 
   describe "#with_published_editions" do
     it "only returns published editions" do
-      create(:guide, slug: "/service-manual/topic-name/1")
-      guide_with_published_editions = create(:published_guide, slug: "/service-manual/topic-name/2")
-      expect(Guide.with_published_editions.to_a).to eq [guide_with_published_editions]
+      guide_community = create(:guide_community, :with_published_edition)
+      create(:guide, slug: "/service-manual/topic-name/1", edition: {
+        content_owner_id: guide_community.id
+      })
+      published_guide = create(:guide, :with_published_edition,
+        slug: "/service-manual/topic-name/2",
+        edition: {
+          content_owner_id: guide_community.id
+        }
+      )
+
+      expect(Guide.with_published_editions.to_a).to eq [guide_community, published_guide]
```

Comment:
Both of these are preexisting..

This is a class method so the describe block text should read `".with_published_editions"`, ie with a dot rather than a hash.

The `to_a` isn't necessary as rspec-rails is smart enough to load the records and compare the arrays rather than tell you that an Array isn't an ActiveRecord::Relation. Also, maybe use `#match_array` rather than `#eq` if we don't care about the order of the results.